src/lib/core/components/visualizations/InputVariables.tsx

```diff
@@ -239,7 +264,7 @@ export function InputVariables(props: Props) {
                   rootEntity={entities[0]}
                   disabledVariables={disabledVariablesByInputName[input.name]}
                   customDisabledVariableMessage={
-                    flattenedConstraints?.[input.name].description
+                    constraints?.[0][input.name].description // just take first description for now
```

Comment:
This code could potentially result in a reference error, e.g., if the array is empty. It might be worth making a helper function to extract the description and to make it a little more defensive. This is an edge case that typescript doesn't handle very well, unfortunately.